clients/clients-integration-tests/src/test/java/org/apache/kafka/clients/admin/ClientTelemetryTest.java

```diff
@@ -159,7 +158,7 @@ private static String[] toArray(List<String>... lists) {
 
     /**
      * We should add a ClientTelemetry into plugins to test the clientInstanceId method Otherwise the
-     * {@link  org.apache.kafka.common.protocol.ApiKeys.GET_TELEMETRY_SUBSCRIPTIONS } command will not be supported
+     * {@link  org.apache.kafka.common.protocol.ApiKeys#GET_TELEMETRY_SUBSCRIPTIONS } command will not be supported
```

Comment:
Since the line is being updated, can we remove the superfluous spaces?

```suggestion
     * {@link org.apache.kafka.common.protocol.ApiKeys#GET_TELEMETRY_SUBSCRIPTIONS} command will not be supported
```